### PR TITLE
Minor Build Configuration Improvements

### PR DESCRIPTION
**Pull request by cloudRoutine**
### gitignore 'tests/fsharpqa/testenv/bin/'

after each build git picks up the detritus

 

this fixes that

### CODE REVIEW

**Comment by cloudRoutine** [comment]
Anyone who has `preview3` or `preivew4` of the dotnet cli installed on their machine can run into issues when they try to build the coreclr version of fsc, or just use the cli within the repo. `global.json` locks the SDK version to prevent those issues.

File: global.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (KevinRansom) [inline global.json]: What are the issues.  The repo should only use the cli found in tools\dotnet/cli.  If we are not including the path to that when we need to then there is a bug.

So please remove this file. the rest looks great.

Comment (cloudRoutine) [inline global.json]: the issue arises is when you want to use `dotnet <command>` in the terminal, like if you want to build, restore, or run only one particular project

Comment (cloudRoutine) [inline global.json]: but the hack to make `dotnet` behave is simple enough too

**Comment by mmitche** [comment]
@dotnet-bot test this please (testing CI move to other server)

**Comment by KevinRansom** [comment]
Thanks @cloudRoutine 

Kevin

**Pull request merged**